tools/generators/files_and_groups/src/Generator/CalculatePathTree.swift

```diff
@@ -175,7 +175,13 @@ extension Generator {
                         children: children
                     )
                 case .generatedFiles(let generatedFiles):
-                    node = .generatedFiles(generatedFiles)
+                    switch generatedFiles {
+                    case .multipleConfigs(let configs):
+                        // Sort configs under same package
+                        node = .generatedFiles(.multipleConfigs(configs.sorted { $0.name < $1.name } ))
```

Comment:
This is the wrong place to do this

Reply:
>Putting the sort here seemed best since other path tree sorting happens here.

Dropped this note in the commit, would you rather see it happen in `calculatePathTree` or inside `CreateInlineBazelGeneratedFiles`? I imagine you're thinking of performance optimizations

Reply:
I would do it up at line 54 above, you can add the sort just before the `map`.

Reply:
Updated